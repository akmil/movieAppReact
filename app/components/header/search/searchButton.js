import React from 'react';
import {Button} from '../../common/button';

export const SearchButton = ({onClick}) => {
    return <div className={'search-button'}>
        <Button onClick={onClick}>search</Button>
    </div>
};
