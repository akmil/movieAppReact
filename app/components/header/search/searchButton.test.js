import React from 'react';
// import { shallow } from 'enzyme';
import {SearchButton} from './searchButton';
import renderer from 'react-test-renderer';

describe('<search SearchButton />', () => {
    it('should render SearchButton without errors', () => {
        const wrapper = renderer.create(<SearchButton/>);
        // const wrapper = shallow(<MainLayout/>);

        expect(wrapper).toMatchSnapshot();
    })
});