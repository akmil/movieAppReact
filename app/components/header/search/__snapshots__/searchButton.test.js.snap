// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<search SearchButton /> should render SearchButton without errors 1`] = `
<div
  className="search-button"
>
  <button
    onClick={undefined}
    style={
      Object {
        "background": "#F44336",
        "border": "none",
        "color": "white",
        "padding": "10px",
      }
    }
  >
    search
  </button>
</div>
`;
